lib/RoadizCoreBundle/src/Console/UsersPasswordCommand.php

```diff
@@ -34,8 +35,14 @@ protected function configure(): void
             )->addOption(
                 'length',
                 'l',
-                InputArgument::OPTIONAL,
+                InputOption::VALUE_OPTIONAL,
                 default: 16,
+            )
+            ->addOption(
+                'plain-password',
+                'p',
+                InputOption::VALUE_OPTIONAL,
```

Comment:
Value is not optional here.
An option is always optional, but here, value is required because you cannot set an empty plain-password

```suggestion
                InputOption::VALUE_REQUIRED,
```